guides/v2.2/magento-functional-testing-framework/release-2/introduction.md

```diff
@@ -26,7 +26,13 @@ Because MFTF tests are written in XML, you no longer need to learn PHP to write
 include note.html
 type="info"
 content="We are actively developing functional tests.
-Refer to `<magento_root>/app/code/<vendor_name>/<module_name>/Test/Mftf/` for examples."
+Refer to `Test/Mftf/` in [magneto modules directories]({{ page.baseurl }}/architecture/archi_perspectives/components/modules/mod_intro.html#where-do-modules-live) for examples."
+%}
+
+{%
+include note.html
+type="info"
+content="At the moment only three modules have MFTF tests: [Sales](https://github.com/magento/magento2/tree/2.2-develop/app/code/Magento/Sales/Test/Mftf), [SalesRule](https://github.com/magento/magento2/tree/2.2-develop/app/code/Magento/SalesRule/Test/Mftf) and [Persistent](https://github.com/magento/magento2/tree/2.2-develop/app/code/Magento/Persistent/Test/Mftf)."
```

Comment:
I may be lacking context, but I'm not sure the above statement is correct:
https://github.com/magento/magento2/tree/2.2-develop/app/code/Magento/Backend/Test/Mftf